<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="start_rest">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="start_top">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="whatsThis">
    <string>вес</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>135</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начало</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>129</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Конец</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_rest">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="start_first_minute">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_first_minute">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_top">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>вес</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Покой</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пик</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1мин.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="start_third_minute">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_third_minute">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="start_fifth_minute">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>480</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_fifth_minute">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3мин.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5мин.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="start_seventh_minute">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>560</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="end_seventh_minute">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>560</y>
     <width>104</width>
     <height>64</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="whatsThis">
    <string>рост</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="autoFormatting">
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7мин.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите данные в соответствующие формы</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
